server/sql_app/main.py

```diff
@@ -187,14 +193,27 @@ def update_study_area(session_id: str, study_area: schemas.StudyArea,
 @app.get("/study_areas/{session_id}", response_model=list[schemas.StudyArea])
 def get_study_areas(session_id: str, db: Session = Depends(get_db)):
     # check that the session exists
+    # TODO: when & why do we need to get the session?
```

Comment:
I *think* I had general thoughts of, should a user have to verify their session as an authentication step before requesting other data.

Reply:
Ah, that's helpful, thanks. I'm not sure I've been consistent with including that or not.